site/content/posts/2022-04-06-april-drumbeat.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "April Drumbeat: Better Output, Faster Debugging, and More Plugins"
+image: /img/sonobuoy.svg
+excerpt: An update about the recent improvements to Sonobuoy which include more support for sidecars, more debugging information surfaced, and unique new plugins. 
+author_name: John Schnake
+author_url: https://github.com/johnschnake
+categories: [kubernetes, sonobuoy, plugins]
+tags: ['Sonobuoy Team', 'John Schnake']
+date: 2022-04-06
+slug: april-drumbeat
+---
+
+# Sonobuoy Drumbeat
+
+We’ve recently switched to releasing Sonobuoy in more rapid, smaller releases and so we wanted to take an opportunity to discuss recent changes more in depth.
+
+## Release Cadence
+
+We’ve moved to releasing Sonobuoy on a weekly basis as long as we have actual code changes. There had been a few instances in the past where features were on the main code branch and being used by developers but not readily available to the wider community and we want to limit that.
+
+## Improved Plugins with Sidecar Containers
+
+Plugins have been able to have sidecar containers ever since the podSpec was configurable. However, they weren’t able to share in some of the features that Sonobuoy provided to the main plugin container; namely automatic environment variables and config-maps.
+
+Recent changes have made it so that if you have a sidecar container, it will have access to the same environment variables that Sonobuoy adds to the main container (e.g. `SONOBUOY_K8S_VERSION`, `SONOBUOY_RESULTS_DIR`, etc).
```

Comment:
Would it be worthwhile to give an example of why this would be useful or will this be obvious to devs?